### PR TITLE
Profile poll status

### PR DESCRIPTION
**Pull request by tobiaserich**

### CODE REVIEW

File: client/src/assets/icons/pendingIcon.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ <svg  xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">
```
Comment (sascharissling) [inline client/src/assets/icons/pendingIcon.svg]: Didnt you get an error with the second xmlns links?

File: client/src/components/DescriptionInput.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const TextInput = styled.textarea`
+  height: 150px;
+  width: 250px;
+  margin-top: 10px;
+  border-radius: 10px;
+  border: none;
+  padding-left: 10px;
+  padding-top: 5px;
+  padding-right: 10px;
```
Comment (sascharissling) [inline client/src/components/DescriptionInput.js]: I'd go with the clockwise format if you have more than two values for padding or margin. 
`padding: 5px 10px 0px 10px ;`

File: client/src/assets/icons/pendingIcon.svg (hunk 1 above)
Comment (tobiaserich) [reply client/src/assets/icons/pendingIcon.svg]: No, but i will check if it is necessary, thanks

File: client/src/components/DescriptionInput.js (hunk 2 above)
Comment (tobiaserich) [reply client/src/components/DescriptionInput.js]: that's a good idea, thanks!

**Pull request merged**